### DISCUSSION TITLE
Downstrema tasks getting skipped

### DISCUSSION THREAD

**Discussion by gunjan075** [apache/airflow#31145] [General]
Downstrema tasks getting skipped after PythonBranoperator
--------------------------------------

My Dag:

start_pipeline >> **check_weekday** >> [task_a,task_b]
task_a >> task_c
task_b >> task_d

**check_weekday** is a BranchPythonOperator
    def check_weekday():
        now = datetime.now()
        if now.weekday() == 0:
            return "run_often_bough_train_data_gen"
        else:
            return "run_often_bough_score_data_gen"

check_weekday = BranchPythonOperator(
        task_id='check_weekday',
        python_callable=check_weekday,
    )
_**check_weekday runs succefully but all downstream task are getting skipped everytime .Need suggestion.**_

**Comment by potiuk** [comment]
Typo in your dag maybe ? Hard to say, you need to debug..
